libraries/AP_HAL_ChibiOS/hwdef/JFB100/hwdef-bl.dat

```diff
@@ -0,0 +1,58 @@
+# hw definition file for processing by chibios_hwdef.py
+# for F765 bootloader
+
+# MCU class and specific type
+MCU STM32F7xx STM32F767xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+# board ID for firmware load
+APJ_BOARD_ID 50
```

Comment:
likely needs a new ID

Reply:
Looks like this was allocated to someone else this morning so maybe increment one more time!

Reply:
I want Request a new ID.